.changeset/twelve-coats-rush.md

```diff
@@ -0,0 +1,35 @@
+---
+'astro': major
+---
+
+The endpoints that have **lower case** name are **deprecated**.
+
+The endpoints methods **needs** to be in **upper case**.
```

Comment:
Plural fixes, plus, we tend to use the lowercase/uppercase variant (one word) in docs, so while this is not incorrect, I would change for consistency.
```suggestion
The endpoints that have **lowercase** names are **deprecated**.

The endpoints methods **need** to be in **uppercase**.
```